docs/library/theodore.md

```diff
@@ -0,0 +1,130 @@
+# Thomson - TO8D (Theodore)
+
+## Background
+
+Theodore is a Thomson TO8D system emulator based on the Daniel Coulom's DCTO8D emulator. The Thomson TO8 is a home computer introduced by French company Thomson SA in 1986. The Thomson TO8D is an improved version that includes a built-in 3.5" floppy drive.
+
+### Author/License
+
+The Theodore core has been authored by
+
+- Thomas Lorblanchès
+
+The Theodore core is licensed under
+
+- [GPLv3](https://github.com/Zlika/theodore/blob/master/LICENSE)
+
+A summary of the licenses behind RetroArch and its cores have found [here](https://docs.libretro.com/tech/licenses/).
+
+## Extensions
+
+Content that can be loaded by the Theodore core have the following file extensions:
+
+- .fd (floppy disk)
+- .k7 (tape)
+- .rom (cartridge)
```

Comment:
I've just added *.m7 to the list of supported extensions, because it is sometimes used for MEMO7 cartridges.